src/diamond/handler/stats_d.py

```diff
@@ -152,7 +154,7 @@ def _connect(self):
             self.connection = statsd.StatsClient(
                 host=self.host,
                 port=self.port
-            )
+            ).pipeline()
```

Comment:
besides backward compatibility? no idea if there is still a good reason nowadays, possibly not